packages/components/pie-form-label/src/index.ts

```diff
@@ -30,6 +35,20 @@ export class PieFormLabel extends RtlMixin(LitElement) implements FormLabelProps
         return optional ? html`<span class="c-formLabel-optional">${optional}</span>` : nothing;
     }
 
+    private handleClick () {
```

Comment:
in a nutshell, if the input component this targets exists, either just focus it or if it can be checked (a check box or switch for example) then check it as well

Reply:
I will need to confirm that the `document` reference doesn't break SSR on consumers